legal-api/src/legal_api/services/filings/validations/annual_report.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Validation for the Annual Report filing."""
+from datetime import timedelta
```

Comment:
should be using https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/utils/datetime.py
```from legal_api.utils.datetime import timedelta```
or fix the base one if necessary

Reply:
Ok. Used timedelta from legal_api.utils.datetime